dont skip drain for unhealthy nodes (#839)
gardener/machine-controller-manager: docs/FAQ.md

Once scaling is achieved, rollout continues.

## How some unhealthy machines are drained quickly ?

If a node is unhealthy for more than the `machine-health-timeout` specified for the `machine-controller`, the controller
health-check moves the machine phase to `Failed`. By default, the `machine-health-timeout` is 10` minutes.

`Failed` machines have their deletion timestamp set and the machine then moves to the `Terminating` phase. The node
drain process is initiated. The drain process is invoked either *gracefully* or *forcefully*.

The usual drain process is graceful. Pods are evicted from the node and the drain process waits until any existing
attached volumes are mounted on new node. However, if the node `Ready` is `False` or the `ReadonlyFilesystem` is `True`

> for greater than `5` minutes, then a forceful drain is initiated. In a forceful drain, pods are deleted

Reviewer: ```suggestion
for greater than `5` minutes(non-configurable), then a forceful drain is initiated. In a forceful drain, pods are deleted
```

Author: done

> and `VolumeAttachment` objects associated with the old node are also deleted. This is followed by the deletion of the

Reviewer: ```suggestion
and `VolumeAttachment` objects associated with the old node are also marked for deletion. This is followed by the deletion of the
```

Author: done

cloud provider VM associated with the `Machine` and then finally ending with the `Node` object deletion. 

> During the deletion of the VM we only delete the local data and boot disks associated with the VM. The disks associated

Reviewer: ```suggestion
During the deletion of the VM we only delete the local data disks and boot disks associated with the VM. The disks associated
```

Author: done
